bundles/asm-policy-v0.0.1/README.md

```diff
@@ -0,0 +1,53 @@
+# asm-policy-v0.0.1
+
+## Description
+Anthos Service Mesh security policy constraints bundle.
+
+## Setters
+
+-   `enforcementAction`: The enforcement mode of the constraints, options are `dryrun` (default) or
```

Comment:
This doesn't need to be a setter. Instead you can use the `set-enforcement-action` [function](https://catalog.kpt.dev/set-enforcement-action/v0.1/).

Reply:
This is using the `set-enforcement-action:v0.1`, here I want to list all the configurable fields, maybe `Setters` to `Configurable fields`?

Reply:
Just give configuration instructions, instead of having a list of configurable fields.

Reply:
+1. Link to the function doc, instead of listing them here because they may change.

Reply:
Would still think have a list of configurable fields would be helpful to show the options and what they for, otherwise it will make the instruction really long as it will need to go through each setters explaining that.